client/src/components/Health/HealthPage.jsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
```

Comment:
same here. These statements could be combined with the DRY principle in mind.

Reply:
Thanks!